### ISSUE TITLE
pass column to onRowClick

### ISSUE THREAD

**Issue by KaptinLin** [icflorescu/mantine-datatable#81]
**Is your feature request related to a problem? Please describe.**
When using onRowClick, the event fired when we click on every column of the row. If we don't want the rowClick fired on some column, we have no idea to do that.

**Describe the solution you'd like**
If we can pass column clicked to onRowClick that would be solved the issue.

```
onRowClick: (data, column) => {
   // check which column we clicked on row
   // do row click
}
```

**Comment by icflorescu** [comment]
Will think about it, but I'd rather implement an additional property for this behaviour, called `onCellClick` or something like that...

**Comment by icflorescu** [comment]
This will be available as soon as `v1.7.8` hits the npm registry.

**Comment by icflorescu** [comment]
Starting with `v1.7.8`, there's an `onCellClick` handler: https://icflorescu.github.io/mantine-datatable/examples/handling-cell-clicks
